roxie/roxiemem/roxiemem.cpp

```diff
@@ -166,6 +170,28 @@ static void initializeHeap(bool allowHugePages, bool allowTransparentHugePages,
 #else
     heapUseHugePages = false;
 
+#ifdef _USE_NUMA
+    int has_numa = numa_available() + 1;
+    if (has_numa)
```

Comment:
(actually I might use >= 0 - it's not clear if other -ve values are ever returned)